server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1733,7 +1805,7 @@ export class AgenticChatController implements ChatHandlers {
         updatedRequestInput.conversationState!.currentMessage!.userInputMessage!.content = content
 
         for (const toolResult of toolResults) {
-            this.#debug(`ToolResult: ${JSON.stringify(toolResult)}`)
+            this.#debug(`ToolResult: ${JSON.stringify(toolResult)} `)
```

Comment:
@jguoamz why add space at the end all of these logs ? same for other cases in this PR.